examples/eks/eks_cluster_optional_readonly/variables.tf

```diff
@@ -0,0 +1,45 @@
+# EKS module variables.
+variable "cluster_name" {
+  type        = string
+  description = "EKS cluster name in AWS account."
+}
+
+variable "cluster_region" {
+  type = string
+  description = "AWS Region in which EKS cluster and supporting resources will be created."
+}
+
+variable "cluster_version" {
+  type        = string
+  description = "EKS cluster version."
+  default     = "1.24"
+}
+
+# Variables required for connecting EKS cluster to CAST AI
+variable "castai_api_token" {
+  type = string
+  description = "CAST AI API token created in console.cast.ai API Access keys section"
+}
+
+variable "castai_api_url" {
+  type        = string
+  description = "CAST AI url to API, default value is https://api.cast.ai"
+  default     = "https://api.dev-master.cast.ai"
+}
+
+variable "delete_nodes_on_disconnect" {
+  type        = bool
+  description = "Optionally delete Cast AI created nodes when the cluster is destroyed"
+  default     = false
+}
+
+variable "aws_assume_role_arn" {
+  type        = string
+  description = "Arn of the role to be used by CAST AI for IAM access"
+  default     = null
+}
+
+variable "readonly" {
+  type = bool
```

Comment:
I would add some description for this variable

Reply:
Updated